css/index.css

```diff
@@ -140,7 +210,58 @@ footer nav {
     font-size: 14px;
 }
 
+footer nav a:hover {
+    background: white;
+    color: black;
```

Comment:
Way to highlight that you can apply animations for both simple and complex methods 👍